File: Java/Medium/CompileError/longest-subarray-with-maximum-bitwise-and_3.java
class Solution {
    public int longestSubarray(int[] nums) {
        
        int n = nums.length;
        int max = 0;
        for (int i = 0; i < n; i++) {
            int cur = nums[i];
            for (int j = i + 1; j < n; j++) {
                cur &= nums[j];
                if (cur == 0) {
                    max = Math.max(max, j - i);
                }
            }
        }
        return max;
    }
}
    }
}

File: Rust/Medium/CompileError/longest-subarray-with-maximum-bitwise-and_3.rs
impl Solution {
    pub fn longest_subarray(nums: Vec<i32>) -> i32 {
        
        let mut ans = 0;
        let mut mask = 0;
        for i in (0..30).rev() {
            mask |= 1 << i;
            let mut st = std::collections::HashSet::new();
            for &num in nums.iter() {
                st.insert(num & mask);
            }
            let tmp = ans | (1 << i);
            for prefix in st {
                if st.contains(&(prefix ^ tmp)) {
                    ans = tmp;
                    break;
                }
            }
        }
        ans
    }
}